help: Document stream notifications mobile feature.

drrosa opened the pull request:
- Documents how to set default notifications for all streams and single streams.

**Screenshots and screen captures:**

- https://zulip.com/help/stream-notifications

![image](https://user-images.githubusercontent.com/2343554/221075289-f2c75763-d031-4309-8654-1ab2f98ebe05.png)


<details>
<summary>Self-review checklist</summary>

- [x] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability

Communicate decisions, questions, and potential concerns.

- [x] Automated tests.

Individual commits are ready for review (see [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/version-control.html)).

- [x] Each commit is a coherent idea.
- [x] Commit message(s) explain reasoning and motivation for changes.

Completed manual review and testing of the following:

- [x] Visual appearance of the changes.
- [x] Links.
</details>

drrosa commented:
@alya This PR is ready for you.

alya commented:
Hm, I think the "Mobile" for the default notifications section should explain how to change the settings from within the mobile app (which you can do under Settings > Notifications).

drrosa commented:
Oh, I missed the **Stream notifications** setting somehow. I fixed the default notifications section and updated the screenshot in the PR description.

alya commented:
Thanks! I think we should clarify what that toggle actually controls, which I believe is just stream notifications for mobile. So maybe something like:

> Toggle **Stream notifications** to configure mobile push notifications for stream messages.

drrosa commented:
> Thanks! I think we should clarify what that toggle actually controls, which I believe is just stream notifications for mobile. So maybe something like:
> 
> > Toggle **Stream notifications** to configure mobile push notifications for stream messages.

Yes, good point! I updated the PR/screenshot.

alya commented:
Works for me, thanks! @timabbott over to you.

timabbott commented:
Merged, thanks @drrosa!
